frontend/src/metabase/containers/FormikForm/FormikForm.tsx

```diff
@@ -92,13 +93,19 @@ function Form<Values extends BaseFieldValues>({
   validate,
   initial,
   normalize,
+  onValuesChange,
   onSubmit,
   onSubmitSuccess,
   ...props
 }: FormContainerProps<Values>) {
   const [error, setError] = useState<string | null>(null);
   const [values, setValues] = useState({});
 
+  const handleValuesChange = (newValues: any) => {
```

Comment:
With this function we enable the Form component to report value changes back to ancestor components.